ansible/roles/etesync/defaults/main.yml

```diff
@@ -0,0 +1,456 @@
+
+                                                                   # ]]]
+# .. envvar:: etesync__http_psk_subpath_enabled [[[
+#
+# Whether EteSync should be deployed on a random subpath that acts as a
+# protection of the web app/API from people not knowing this PSK.
+# For a discussion in which scenarios this can make sense, refer to
+# `RFC: Support subpath/subdir hosting for additional security`__.
+#
+# .. __: https://github.com/debops/debops/issues/1233
+etesync__http_psk_subpath_enabled: False
+
+                                                                   # ]]]
+# .. envvar:: etesync__http_psk_subpath [[[
+#
+# PSK used as subpath that acts as the first layer of defense in a security in
+# depth concept if enabled.
+etesync__http_psk_subpath: '{{ lookup("password", secret + "/etesync/" +
+                                 inventory_hostname + "/config/subpath chars=ascii_letters,digits length=23")
+                               if etesync__http_psk_subpath_enabled|bool
+                               else "" }}'
+
+                                                                   # ]]]
+# .. envvar:: etesync__url [[[
+#
+# The URL where the EteSync server will be reachable. Exposed as variable here
+# as you might want to use it in your custom user password lookup for
+# integrating into your password manager.
+etesync__url: '{{ "https://" + etesync__fqdn + "/" + etesync__http_psk_subpath }}'
+
+                                                                   # ]]]
+# .. envvar:: etesync__admin_auth_basic_realm [[[
+#
+# A string which will be displayed as the realm in the browser user/password
+# dialog box during HTTP Basic Authentication for the admin interface.
+etesync__admin_auth_basic_realm: 'Access to EteSync admin interface is restricted'
+
+                                                                   # ]]]
+# .. envvar:: etesync__admin_auth_basic_filename [[[
+#
+# Absolute path to the file that contains usernames and passwords for HTTP
+# Basic Authentication for the admin interface.
+etesync__admin_auth_basic_filename: ''
+
+                                                                   # ]]]
+# Configuration variables for other Ansible roles [[[
+# ---------------------------------------------------
+
+# .. envvar:: etesync__keyring__dependent_gpg_keys [[[
+#
+# Configuration for the :ref:`debops.keyring` Ansible role.
+etesync__keyring__dependent_gpg_keys:
+
+  - user: '{{ etesync__user }}'
+    group: '{{ etesync__group }}'
+    home: '{{ etesync__home }}'
+    id: '{{ etesync__git_gpg_key_id }}'
+
+                                                                   # ]]]
+# .. envvar:: etesync__python__dependent_packages3 [[[
+#
+# Configuration for the :ref:`debops.python` Ansible role.
+etesync__python__dependent_packages3:
+
+  - 'python3-setproctitle'
+
+  - 'python3-dev'
+
+  ## Django>=2.1.7,<2.1.999 is required which is currently only in Debian sid.
+  # - 'python3-django'
```

Comment:
Apt pinning?

Reply:
Please refer to https://docs.debops.org/en/master/dep/dep-0003.html Debian sid is not enabled on production servers.

Reply:
oh yeah. Also DEP-11 is probably ambiguous with debian appstream.

Reply:
Sid packages are not required obviously. I just tried to install as much of the dependencies from stable but then had to give up. I use virtualenv and pip for now.